src/commands/options.ts

```diff
@@ -0,0 +1,18 @@
+import {Command} from '@oclif/command'
+
+// import Help from '..'
+
+export default class OptionsCommand extends Command {
```

Comment:
this would create a new command for all the CLIs which we definitely do not want. Inline the class like we do here: https://github.com/oclif/plugin-help/blob/master/test/command.test.ts#L116

Reply:
Makes sense to me now :D I updated the tests